# HG changeset patch
# User marco atzeri <marco.atzeri@gmail.com>
# Date 1380899100 25200
#      Fri Oct 04 08:05:00 2013 -0700
# Node ID ddecb2e231d473eac16d9fcf8fe28ed195908797
# Parent  234731d524c276ea0163f9d93f1b7e567372173e
configure.ac: Search for rcc variants on Cygwin to build GUI.

* configure.ac: Search for rcc variants on Cygwin to build GUI.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2639,17 +2639,17 @@ if test $build_gui = yes; then
       AC_MSG_WARN([Qt >= 4.0.0 not found -- disabling GUI])
       build_gui=no
     fi
   fi
 
   if test $build_gui = yes; then
     AC_CHECK_TOOLS(MOC, [moc-qt5 moc-qt4 moc])
     AC_CHECK_TOOLS(UIC, [uic-qt5 uic-qt4 uic])
-    AC_CHECK_TOOLS(RCC, [rcc])
+    AC_CHECK_TOOLS(RCC, [rcc-qt5 rcc-qt4 rcc])
     AC_CHECK_TOOLS(LRELEASE, [lrelease-qt5 lrelease-qt4 lrelease])
     if test -n "$MOC" && test -n "$UIC" && test -n "$RCC" && test -n "$LRELEASE"; then
       AC_DEFINE(HAVE_QT, 1, 
         [Define to 1 if Qt is available (libraries, developer header files, utility programs (moc, uic, rcc, and lrelease))])
     else
       AC_MSG_WARN([Qt utility programs moc, uic, rcc, and lrelease not found -- disabling GUI])
       build_gui=no
     fi
